test/org/sagebionetworks/bridge/services/AuthenticationServiceImplTest.java

```diff
@@ -71,12 +99,12 @@ public void signInNoUsername() throws Exception {
 
     @Test(expected = BridgeServiceException.class)
     public void signInNoPassword() throws Exception {
-        authService.signIn(testUser.getStudy(), new SignIn("foo", null));
+        authService.signIn(testUser.getStudy(), new SignIn(TEST_DATA_GROUP, null));
```

Comment:
I think you were a little overzealous with your find and replace. SignIn takes a username not a data group. (Here and in signInInvalidCredentials() below.